h/cli/commands/render_annotation_text.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
```

Comment:
Perhaps this should be a data migration?

I know why you've added it here (in case we change the rendering algorithm in the future and need to rerun it) but I actually think even then putting it in a data migration has the advantage that it makes it explicit in the repo history that the format has changed and we need to update the database. It's not hard to retrieve a migration from the past and create a new one with the same content. And it means we don't end up with an unbounded number of these one-off commands in the CLI.